Add gimbal device id param to gimbal_manager_set_attitude

coderkalyan opened the pull request:
the GIMBAL_MANAGER_SET_ATTITUDE messsage is missing a parameter to specify gimbal device ID, this PR adds it.

julianoes commented on message_definitions/v1.0/common.xml:
```diff
@@ -6077,6 +6077,7 @@
       <field type="uint8_t" name="target_system">System ID</field>
       <field type="uint8_t" name="target_component">Component ID</field>
       <field type="uint32_t" name="flags" enum="GIMBAL_MANAGER_FLAGS">High level gimbal manager flags to use.</field>
+      <field type="uint8_t" name="gimbal_device_id">Component ID of gimbal device to address (or 1-6 for non-MAVLink gimbal), 0 for all gimbal device components. (Send command multiple times for more than one but not all gimbals.)</field>
```
In `GIMBAL_INFORMATION` we call it `gimbal_component`. It would probably be better to have them consistent. Which one is better?

coderkalyan replied:
@julianoes In my opinion, gimbal device id, or even just gimbal_id, because we cannot assume that the gimbal device is even a component - the manager provides a wrapper for PWM or other gimbals as well...

coderkalyan commented:
Update: @julianoes I renamed `gimbal_component` to `gimbal_device_id` in `GIMBAL_INFORMATION`. I also added this field to the `GIMBAL_MANAGER_STATUS` message, will probably be useful to know which gimbal manager the status belongs to :thinking: :smile: 

I also renamed `angular rate` to `angular velocity`.

julianoes commented:
@olliw42 this makes sense, right?

olliw42 commented:
hey folks

sorry for being so totally unresponsive to this mavlink gimbal stuff lately. I just want to quickly state, but really want to state:

You should NOT think that this is because I would have lost interest, to the contrary!

It's just that I don't have the concentration to look at it with sufficient care to make useful comments. And the reason for why I lack the concentration is because I'm heavily working on getting to a point there I actually can implement AND use this stuff in real world! (you know that QGC turned out to not be an efficient path to progress forward, so I had to establish a substitute which would work for me, and it's where my time goes into currently). That is, even though I'm silent on the topic now, I - hopefully - will come back with experience from practical usage - which I believe might be more helpful than just theories :):):)

So, please continue to keep me in the line as you did here, it's very much appreciated, and please don't be upset about my silence :)

cheers, and all the best,
Olli

julianoes commented:
@olliw42 phew, good to know! Thanks for the heads-up and happy Easter!
